lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -372,6 +388,17 @@ def _user_certificate_info(self, user, context, course_grade, bulk_certs):
         )
         return certificate_info
 
+    def _enrollment_status(self, user, context):
+        """
+        Returns a list of a string that indicates whether a user
+        is actively enrolled in a course.
+        """
+        if not getattr(self, '_enrollment_status_by_user', None):
+            self._enrollment_status_by_user = _user_enrollment_status_mapping(context.course_id)
+        if self._enrollment_status_by_user.get(user.id):
+            return [ENROLLED_IN_COURSE]
+        return [NOT_ENROLLED_IN_COURSE]
```

Comment:
My only New Year's resolution was to stop using one-line conditional expressions :)